azrcrv-redirect.php

```diff
@@ -385,7 +387,10 @@ function display_options() {
 
 				<div id="tabs">
 					<div id="tab-panel-1" >
-						<?php echo $tab_1; ?>
+						<?php
+						// String prepared above is safe.
+						// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+						echo $tab_1; ?>
```

Comment:
There is some conditional logic involved in building these `$tab_1` and similar strings but they are ultimately only used once. I think it might lead to cleaner code to move that logic here and just output all the stuff in the `$tab_1` string directly.

This is an example of a coding practice that (in my opinion) should be left up to the plugin developer, but it should be understood that when the coding standards checks flag this kind of issue then the plugin reviewers will probably ask about it and the review process could take longer.